index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Linus Shaddai Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <img src="./img/Logo Placeholder.svg" alt="logo" />
+        <img class="hamburger-menu" src="./img/Union.svg" alt="hambuger menu" />
+      </nav>
+    </header>
+
+    <section class="headline">
+      <div class="headline-content">
+        <h1>I'm Shaddai,<br />Glad to see you</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <a href="#" class="connect">Lets Connect</a>
+        <div class="headline-socials">
+          <a href=""><img src="./img/Vector.svg" alt="" /></a>
+          <a href=""><img src="./img/Linkedin icon.svg" alt="" /></a>
+          <a href=""><img src="./img/Vector (1).svg" alt="" /></a>
+          <a href=""><img src="./img/Vector (2).svg" alt="" /></a>
+          <a href=""><img src="./img/Vector (3).svg" alt="" /></a>
+        </div>
```

Comment:
- [ ] Using an unordered list `ul` here instead of a `div` will sound great for accessibility's sake.. What do you think?
- [ ] Kindly make sure your anchors tag are grabbed inside a `li`